src/components/preferences/Preferences.js

```diff
@@ -340,10 +423,42 @@ function Preferences(props) {
         }
         return errors;
     };
+
+    const mapPropsToValues = (bootstrap) => {
+        if (bootstrap === null || bootstrap === undefined) {
+            return {};
+        }
+        if (bootstrap?.preferences) {
+            if (bootstrap?.apps_info?.webhooks[0]) {
+                let hook = bootstrap?.apps_info?.webhooks[0];
+                const selectedTopics = hook?.topics;
+                const reducer = (acc, currVal) => ({
+                    ...acc,
+                    [currVal]: selectedTopics?.includes(currVal),
+                });
+                const topics = selectedTopics.reduce(reducer, {});
```

Comment:
This should be reducing on all possible topics:
```suggestion
                const topics = webhookTopics.reduce(reducer, {});
```

Reply:
Oops! Yes.

Reply:
Heh..Actually there is little more to it.  line `437` should be:
` const reducer = (acc, currVal) => ({
                    ...acc,
                    [currVal?.topic]: selectedTopics?.includes(currVal?.topic),
                });`